### ISSUE TITLE
How to silence console.log when reading a zip?

### ISSUE THREAD

**Issue by biplobsd** [rinsuki/async-zip-reader#3]
![image](https://github.com/rinsuki/async-zip-reader/assets/43641536/7d6557ae-a820-42b4-bb2c-2d3d78220def)

Is there any option to silence these console.logs?
https://github.com/rinsuki/async-zip-reader/blob/0048f85b0a6a50b69eeaced687a7fc728e7b9819/src/entry.ts#L44
https://github.com/rinsuki/async-zip-reader/blob/0048f85b0a6a50b69eeaced687a7fc728e7b9819/src/reader.ts#L95
https://github.com/rinsuki/async-zip-reader/blob/0048f85b0a6a50b69eeaced687a7fc728e7b9819/src/reader.ts#L91

Because these logs make it hard to find other important logs.

**Comment by rinsuki** [comment]
Released as 0.2.0 (now logging is disabled as default, you can enable by pass `{debugLogging: true}` to new argument of `ZipReader.init` (e.g. `ZipReader.init(provider, "shift-jis", {debugLogging: true}`)

**Comment by biplobsd** [comment]
@rinsuki Great, Thank you for the release.

I used your library in my [YouTube subscription transfer](https://github.com/biplobsd/yst#google-takeout) project to extract Google Takeout zip files. It's so simple to use and async, Great work 🔥🔥🚀💯

Before, I used `JSZip.loadAsync`, but it blocked the main thread and made the UI hang. Your library is a great solution for this issue.

Thank you for your excellent work!
